src/components/Map/USACountyMap.tsx

```diff
@@ -1,87 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
 import { ComposableMap, Geographies, Geography } from 'react-simple-maps';
 import { geoAlbersUsaTerritories } from 'geo-albers-usa-territories';
+import ReactTooltip from 'react-tooltip';
 import { colorFromLocationSummary } from 'common/colors';
 import { useSummaries } from 'common/location_summaries';
 import { ScreenshotReady } from 'components/Screenshot';
 import regions, { State as StateType } from 'common/regions';
+import stateGeographies from 'common/data/states-10m.json';
```

Comment:
Out of curiosity, why not make this async too?  Would save another 35k of gzipped JSON that has to be downloaded/parsed/rendered-as-svg before FCP / LCP can happen, right?

Reply:
The rendering of the map is not ideal while the data loads - I want to try having a good loading state to minimize the impact of loading the states async first. I'm going to try that tomorrow :)